content/docs/components/hyperparameter-tuning/env-variables.md

```diff
@@ -0,0 +1,169 @@
++++
+title = "Katib components environment variables"
```

Comment:
The website pattern is to use title case for page headings. I'd recommend also that you swap the word order to make the meaning clearer:

```
Environment Variables for Katib Components
```